components/lists/lists.css

```diff
@@ -0,0 +1,87 @@
+@import "../design-guide.css";
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
You can avoid to use in every CSS file by simply add it in main file.

Reply:
Done
Fixed in later commits